tmt/base.py

```diff
@@ -332,6 +332,46 @@ def _export(
     )
 
 
+def create_adjust_callback(logger: tmt.log.Logger) -> fmf.base.AdjustCallback:
+    """
+    Create a custom callback for fmf's adjust.
+
+    Given the ``adjust`` rules are applied on many places, for
+    proper logging they need their own specific logger. Create
+    a callback closure with the given logger.
+    """
+
+    def callback(
+            node: fmf.Tree,
+            rule: _RawAdjustRule,
+            applied: Optional[bool]) -> None:
+        if applied is None:
+            logger.verbose(
+                f"Adjust rule **skipped** on '{node.name}'",
```

Comment:
the UX looks weird to me, could we just color the outcome of the adjust rule apply maybe? And ideally without **, it looks odd, I believe it would be enought to mark the word:

I would leave the whole text in `cyan`, while:

* green - applied
* red - not applied
* blue (or white) - skipped

Reply:
@happz kind ping on this comment

Reply:
Colors are there to help on the console, they should not be the primary information carrier. I don't care about that enough to not change them.

I can also drop `**`, not crucial, but helpful in highlighting the decision when there are no colors, e.g. when the output is piped into a file. This, I believe, will be most of the time, as there's plenty of `adjust` around & searching a file is much easier than following the output on the terminal.

I don't understand whether changing colors and dropping `**` would make the UX less weird in your eyes, or whether is there anything else left to tweak.

Reply:
Agree with @thrix that `**` looks sort of disturbing and I'd say in non-color view search highlight helps much more. I propose to use `cyan` for keys instead of `green` as the green color almost everywhere causes it hard to actually see the green applied rules. @happz, are you ok with 5917b59b?